homeassistant/components/templated_automation/__init__.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Templated Automation.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/templated_automation
+"""
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_SOURCE, EVENT_STATE_CHANGED, STATE_OFF, \
+    SERVICE_TURN_OFF, SERVICE_TURN_ON, ATTR_ENTITY_ID, ATTR_STATE
+
+CONF_BIDIRECTIONAL = 'bidirectional'
+CONF_INVERSE = 'inverse'
+CONF_TARGET = 'target'
+
+DEFAULT_BIDIRECTIONAL = False
+DEFAULT_INVERSE = False
+
+DOMAIN = 'templated_automation'
```

Comment:
I'm not super sold on the name. It's vague to me what that means. But I don't know anything better yet.

Reply:
Neither am I. Totally open to other ideas.